tests/cover/test_type_lookup.py

```diff
@@ -28,7 +28,7 @@
 # Build a set of all types output by core strategies
 blacklist = [
     'builds', 'iterables', 'permutations', 'random_module', 'randoms',
-    'runner', 'sampled_from', 'streaming',
+    'runner', 'sampled_from', 'streaming', 'fspaths',
```

Comment:
This is entirely correct, and makes me want to register a strategy for `os.PathLike`.  In `hypothesis/searchstrategy/types.py`, add the following at line 131 (and `import os` at the top):

    if hasattr(os, 'PathLike'):  # pragma: no branch
        _global_type_lookup[os.PathLike] = st.fspaths(allow_pathlike=False).map(st._PathLike)